src/Symfony/Component/HttpKernel/DataCollector/Util/ValueExporter.php

```diff
@@ -33,6 +33,7 @@ public function exportValue($value, $depth = 1, $deep = false)
 
         if (is_object($value)) {
             if ($value instanceof \DateTime || $value instanceof \DateTimeInterface) {
+                // note: \DateTime::ISO8601 is not ISO8601 compatible; \DateTime::ATOM IS ISO8601 compatible
                 return sprintf('Object(%s) - %s', get_class($value), $value->format(\DateTime::ISO8601));
```

Comment:
That code is in a DataCollector, so there is no BC promise here. So, you can update the code.